src/tests/UnsInternal.test.ts

```diff
@@ -0,0 +1,387 @@
+import UnsInternal from '../UnsInternal';
```

Comment:
@sudoryan Add test for each UnsInternal getter that check's the following corner cases:
- Domain doesn't exits (is not registered)
- Domain exists, but doesn't have records

Thanks.